pkg/storage/trb_request_attendee.go

```diff
@@ -88,17 +114,42 @@ func (s *Store) GetTRBRequestAttendeesByTRBRequestID(ctx context.Context, trbReq
 }
 
 // DeleteTRBRequestAttendee deletes an existing TRB request attendee record in the database
-func (s *Store) DeleteTRBRequestAttendee(ctx context.Context, id uuid.UUID) error {
-	const deleteTRBRequestAttendeeSQL = `
+func (s *Store) DeleteTRBRequestAttendee(ctx context.Context, id uuid.UUID) (*models.TRBRequestAttendee, error) {
+	stmt, err := s.db.PrepareNamed(`
 		DELETE FROM trb_request_attendees
-		WHERE id = $1;`
-
-	_, err := s.db.Exec(deleteTRBRequestAttendeeSQL, id)
+		WHERE id = $1
+		RETURNING
+			:id,
```

Comment:
this returning should have all fields from the database table, or there will be blanks / errors in the GQL

![image](https://user-images.githubusercontent.com/99294987/192340349-19b166e5-14b2-492a-9599-b03617fc380c.png)